### ISSUE TITLE
Plugin built using a newer version of Java

### ISSUE THREAD

**Issue by Xerus2000** [BreadMoirai/github-release-gradle-plugin#4]
My project still uses Java 8 and thus I encountered this issue after updating the plugin
```
A problem occurred configuring root project 'MonsterUtilities'.
> Failed to notify project evaluation listener.
   > com/github/breadmoirai/CachedProvider has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```
Is there a reason to build the plugin with Java 10?

**Comment by BreadMoirai** [comment]
whoops. I will fix that shortly.

**Comment by Xerus2000** [comment]
Hey, little reminder to not forget this ;)

**Comment by BreadMoirai** [comment]
Should be fixed with version 2.2.1
